sycl/source/CMakeLists.txt

```diff
@@ -184,28 +197,6 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
     target_compile_definitions(${LIB_OBJ_NAME} PRIVATE SYCL_EXT_JIT_ENABLE)
   endif(SYCL_ENABLE_EXTENSION_JIT)
 
-  find_package(Threads REQUIRED)
-
-  target_link_libraries(${LIB_NAME}
-    PRIVATE
-      ${CMAKE_DL_LIBS}
-      ${CMAKE_THREAD_LIBS_INIT}
-  )
-
-  # Link UR
-  target_link_libraries(${LIB_NAME}
-    PRIVATE
-      UnifiedRuntime-Headers
-      UnifiedRuntimeCommon
-  )
```

Comment:
Do we don't want to link `UnifiedRuntime-Headers`, `UnifiedRuntimeCommon` libraries? I don't see linking these in the code above:
```
target_link_libraries(${LIB_NAME}
    PRIVATE
      ${CMAKE_DL_LIBS}
      ${CMAKE_THREAD_LIBS_INIT}
      $<$<NOT:$<BOOL:${WIN32}>>:UnifiedRuntimeLoader>
      $<$<BOOL:${WIN32}>:shlwapi>
  )
```

Reply:
UR headers is an `INTERFACE` library - it only provides include directories and since `LIB_NAME` is a link-only target, we do not care about include directories there. Note that we still "link" that interface library to `LIB_OBJ_NAME` target, see #17413 and review comments there (similar question).

UR common seems to be an internal library which shouldn't be used by anyone outside of the UR. Once again, see the aforementioned #17413 and #17388 for confirmations from UR folks that we shouldn't use their common library